File: week_1_assignment/cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int change;
    do
    {
        change = get_int("Change owed: ");
    }
    while (change < 0);

    int coins = 0;

    while (change >= 25)
    {
        change -= 25;
        coins++;
    }

    while (change >= 10)
    {
        change -= 10;
        coins++;
    }

    while (change >= 5)
    {
        change -= 5;
        coins++;
    }

    while (change >= 1)
    {
        change -= 1;
        coins++;
    }

    printf("%i\n", coins);
}

File: week_6_assignment/sentimental-cash/cash.py
# TODO

from cs50 import get_float

while True:
    c = get_float("Change: ")
    if c > 0:
        break

c = round(c * 100)

count = 0

while c >= 25:
    c = c - 25
    count += 1

while c >= 10:
    c = c - 10
    count += 1

while c >= 5:
    c = c - 5
    count += 1

while c >= 1:
    c = c - 1
    count += 1

print("Total coins: ", count)
